scrapy/commands/__init__.py

```diff
@@ -79,7 +80,8 @@ def process_options(self, args, opts):
             self.settings.setdict(arglist_to_dict(opts.set),
                                   priority='cmdline')
         except ValueError:
-            raise UsageError("Invalid -s value, use -s NAME=VALUE", print_help=False)
+            raise UsageError(
+                "Invalid -s value, use -s NAME=VALUE", print_help=False)
```

Comment:
No need to change this line. In Scrapy, even though aiming for 79-character-long lines, style changes to existing code because of line length should be avoided as long as line length is within our hard limit: 119 characters.

It also applies to a similar line in `crawl.py`.